code/modules/atmospherics/gasmixtures/reaction_factors.dm

```diff
@@ -0,0 +1,83 @@
+/datum/gas_reaction/water_vapor/init_factors()
+	factor = list(
+		/datum/gas/water_vapor = "Condensation will consume 0.25 moles, freezing will not consume any",
```

Comment:
i know is a pain but i think the numbers and amounts should not be hardcoded in but they should be calculated in some way so that if you change the reaction these would automatically import the new values

Reply:
I agree. Which is why Im waiting for temp's reaction cleanup pr to get in first.